docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,55 @@
+[[ml-dataframes]]
+=== {dataframes-cap}
+
+beta[]
+
+A _{dataframe}_ is a transformation of a dataset by certain rules. You can think 
+of it like a spreadsheet or a data table that makes your data ready to be analyzed 
+and organized.
+
+A lot of {es} datasets are organized as a stream of events: each event is a individual 
+document, e.g. a single item purchase. {dataframe-cap} pivots allow you to summarize 
+this data, bringing it into an organized, more analysis friendly format, e.g. 
+summarizing all purchases of a single customer (see the example below).
+
+The {dataframe} feature enables you to define a _pivot_ which is a set of features 
+that transform the dataset into a different, more digestible format. Pivoting 
+results in a summary of your dataset (which is the {dataframe} itself).
+
+Defining a pivot consist of two main parts. First, you select one or more fields 
+that your dataset will be grouped by. Principally you can select categorical 
+fields (terms) for grouping. You can also select numerical fields, in this case, 
+the field values will be bucketed using an interval you specify.
+
+The second step is deciding how you want to aggregate the grouped data. When 
+using aggregations, you practically ask questions about the dataset. There are 
+different types of aggregations, each with its own purpose and output. You can 
+learn more about the supported aggregations and group-by fields 
+{ref}/data-frame-transform-pivot.html[here].
```

Comment:
I think it's a good idea to have the title of the destination page in the link, so that people can decide whether they want to follow it (and in case something goes wrong with the link and they need to search for it by its title).
```suggestion
{ref}/data-frame-transform-pivot.html[Pivot resources].
```